test/nsidc-icesat2/NSIDC-ICESAT2_Regression.ipynb

```diff
@@ -0,0 +1,433 @@
+    "        },\n",
+    "        'ATL09': {\n",
+    "            'collection_concept_id': Collection(id='C1256563776-NSIDC_CUAT'),\n",
+    "            'granule_id': 'G1262106425-NSIDC_CUAT',\n",
+    "            'shape': 'ancillary/Tasmania_sliver.geojson',\n",
+    "        },\n",
+    "        'ATL10': {\n",
+    "            'collection_concept_id': Collection(id='C1256535487-NSIDC_CUAT'),\n",
+    "            'granule_id': 'G1261681735-NSIDC_CUAT',\n",
+    "            'shape': 'ancillary/Ross_Sea_positive_lon_only.geojson',\n",
+    "        },\n",
+    "    },\n",
+    "}\n",
+    "\n",
+    "\n",
+    "environment_configuration = {\n",
+    "    'https://harmony.uat.earthdata.nasa.gov': {\n",
+    "        **non_production_configuration,\n",
+    "        'env': Environment.UAT,\n",
+    "    },\n",
+    "    'https://harmony.sit.earthdata.nasa.gov': {\n",
+    "        **non_production_configuration,\n",
+    "        'env': Environment.SIT,\n",
+    "    },\n",
+    "    'http://localhost:3000': {\n",
+    "        **non_production_configuration,\n",
+    "        'env': Environment.LOCAL,\n",
+    "    },\n",
+    "}\n",
+    "\n",
+    "configuration = environment_configuration.get(harmony_host_url)\n",
+    "\n",
+    "if configuration is not None:\n",
+    "    harmony_client = Client(env=configuration['env'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b115ba16-0d31-43b0-9f57-6a7061b7eeda",
+   "metadata": {},
+   "source": [
+    "### Run Bounding Box Tests"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e0788c88-6aee-450a-be35-087ba36a1d80",
+   "metadata": {},
+   "source": [
+    "The next cell runs through each of the subset by bounding box tests forming requests that are submitted to Harmony and comparing the downloaded results against reference data files that have been verified.  This ensures that Harmony continues to return the expected binary files for expected requests."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e745045b-10fc-4e6c-808c-3bdaf2725d03",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "test_name = 'subset_bounding_box'\n",
+    "with TemporaryDirectory() as tmp_dir:\n",
+    "    if configuration is not None:\n",
+    "        for shortname, test_config in configuration[test_name].items():\n",
+    "            test_request = Request(\n",
+    "                collection=test_config['collection_concept_id'],\n",
+    "                granule_id=[test_config['granule_id']],\n",
+    "                spatial=test_config['spatial'],\n",
+    "            )\n",
+    "            test_output = tmp_dir / Path(f'{shortname}_{test_name}.h5')\n",
+    "            test_reference = Path(\n",
+    "                f'reference_files/{test_output.stem}_reference{test_output.suffix}'\n",
+    "            )\n",
+    "\n",
+    "            submit_and_download(harmony_client, test_request, test_output)\n",
```

Comment:
@flamingbear I got an error here when running locally: 

`OSError: [Errno 18] Invalid cross-device link: '4720896_ATL03_20200427193622_04930702_006_02_subsetted.h5' -> '[/tmp/tmp3bpwuc_y/ATL03_subset_bounding_box.h5](https://openscapes.2i2c.cloud/tmp/tmp3bpwuc_y/ATL03_subset_bounding_box.h5)'`

Reply:
This is probably because you're not actually local, but when I'm back I'll see if [this stackoverflow](https://stackoverflow.com/a/43967659/66100) answer can fix this for you

Reply:
So I think this 89cae89 might fix it. It isn't hurting the existing local and docker runs. If you get a chance you can update your branch and try again? 🙇

Reply:
Looks like that did the trick. Notebook is running end to end now in the Openscapes Hub. 🎉